netbench/netbench-driver/src/bin/netbench-driver-s2n-quic-client.rs

```diff
@@ -21,7 +22,7 @@ pub struct Client {
     opts: netbench_driver::Client,
 
     #[structopt(long)]
```

Comment:
Structopt will read env vars for you

```suggestion
    #[structopt(long, env = "DISABLE_GSO")]
```